scripts/reset-state.sh

```diff
@@ -18,6 +18,7 @@ elif [ "$platform" == "Linux" ]; then
 
     dirs=(
         "$HOME/.radicle"
+        "$HOME/.config/Radicle Upstream"
         "$config_home/radicle"
         "$config_home/radicle-upstream"
```

Comment:
This is obsolete now.

Reply:
Right. Should this perhaps be `"$config_home/Radicle Upstream"` and the one on L21 be deleted?

Reply:
yus

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1325/commits/d641c203dab85cb2d918feba4b5c922e50176ec9